argocd/setup.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2022-present
```

Comment:
I think this file can be removed given that it's py3 only.

Reply:
TIL

Reply:
hmm, the file is still there though. Do we need it in the end?

Reply:
Sorry missed it.